clang/tools/scan-build-py/CMakeLists.txt

```diff
@@ -167,11 +167,11 @@ if (SYCLomatic_CUSTOMIZATION)
     )
 
     install(TARGETS intercept-stub
-            DESTINATION lib/libear
+            DESTINATION ${CMAKE_BINARY_DIR}/lib/libear
```

Comment:
This line changes original logic, which make the intercept-stub only installed in the build directory, not the installation directory, which should be the root cause of CI test failure.

Reply:
Done.